sbin/system_calls.h

```diff
@@ -0,0 +1,48 @@
+#ifndef _SYS__CALL_H_
+#define _SYS__CALL_H_
+
+#include <stdint.h>
+
+#define SYS_EXIT 0
+#define SYS_CLOSE 1
+#define SYS_ENV 2
+#define SYS_EXEC 3
+#define SYS_FORK 4
+#define SYS_FSTAT 5
+#define SYS_GETPID 6
+#define SYS_ISATTY 7
+#define SYS_KILL 8
+#define SYS_LINK 9
+#define SYS_LSEEK 10
+#define SYS_OPEN 11
+#define SYS_READ 12
+#define SYS_STAT 13
+#define SYS_UNLINK 14
+#define SYS_WAIT 15
+#define SYS_WRITE 16
+#define SYS_TIME 17
+
+extern void _exit();
```

Comment:
We shouldn't need to define the externs I think. I should be able to link libc with system_calls.S.